### DISCUSSION TITLE
Possible to show ratings for all positions regardless of actual player position?

### DISCUSSION THREAD

**Discussion by mcbgun** [olimorris/PyScoutFM#8] [Ideas]
Hi,

In addition to our other moneyball discussion....just thought it would be an idea to see ratings for all positions for a player in case you'd want to retrain them...for instance I have a midfielder who using Squirrel's tool showed good ratings for a wing back. Just a thought...does it work with your model?

**Comment by olimorris** [comment]
Very good point. I can probably add a separate flag for this from the command line
